runtime/jcl/common/thread.cpp

```diff
@@ -280,15 +280,20 @@ Java_java_lang_Thread_interruptImpl(JNIEnv *env, jobject rcv)
 	vmFuncs->internalEnterVMFromJNI(currentThread);
 	j9object_t receiverObject = J9_JNI_UNWRAP_REFERENCE(rcv);
 	J9VMThread *targetThread = J9VMJAVALANGTHREAD_THREADREF(currentThread, receiverObject);
-	if (J9VMJAVALANGTHREAD_STARTED(currentThread, receiverObject)) {
-		if (NULL != targetThread) {
+	if (NULL != targetThread) {
```

Comment:
If && is used here, the code below doesn't need to be duplicated.

Reply:
Agreed and replaced with `&&` (duplicated code removed).